src/CMakeLists.txt

```diff
@@ -16,16 +16,16 @@ set (sources
 )
 
 set (gtest_sources
-  Application_TEST
```

Comment:
This changes are related to this PR https://github.com/gazebosim/gz-gui/pull/428